src/Resources/app/storefront/src/checkout/confirm-order.plugin.js

```diff
@@ -127,8 +126,17 @@ export default class ConfirmOrderPlugin extends Plugin {
     afterPayOrder(orderId, response) {
         try {
             response = JSON.parse(response);
-            window.returnUrl = response.redirectUrl;
+            this.returnUrl = response.redirectUrl;
+        } catch (e) {
+            console.log(e);
+            return;
+        }
+
+        if (this.returnUrl == this.errorUrl.toString()) {
+            location.href = this.returnUrl;
```

Comment:
@peterojo Could you add a comment here to easily understand what happened?

Reply:
Done.